vm/src/builtins/property.rs

```diff
@@ -1,23 +1,24 @@
 /*! Python `property` descriptor class.
 
 */
-use super::{PyType, PyTypeRef};
+use super::{PyStrRef, PyType, PyTypeRef};
 use crate::common::lock::PyRwLock;
-use crate::function::PosArgs;
+use crate::function::{IntoFuncArgs, PosArgs};
 use crate::{
     class::PyClassImpl,
     function::{FuncArgs, PySetterValue},
     types::{Constructor, GetDescriptor, Initializer},
     AsObject, Context, Py, PyObject, PyObjectRef, PyPayload, PyRef, PyResult, VirtualMachine,
 };
 
-#[pyclass(module = false, name = "property", traverse)]
+#[pyclass(module = false, name = "property")]
```

Comment:
```suggestion
#[pyclass(module = false, name = "property", traverse)]
```
`traverse` need to be kept for GC